x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/classification_exploration.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment, useState, useEffect } from 'react';
+import { EuiCallOut, EuiPanel, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { ml } from '../../../../../services/ml_api_service';
+import { DataFrameAnalyticsConfig } from '../../../../common';
+import { EvaluatePanel } from './evaluate_panel';
+import { ResultsTable } from './results_table';
+import { DATA_FRAME_TASK_STATE } from '../../../analytics_management/components/analytics_list/common';
+import { ResultsSearchQuery, defaultSearchQuery } from '../../../../common/analytics';
+import { LoadingPanel } from '../loading_panel';
+
+interface GetDataFrameAnalyticsResponse {
+  count: number;
+  data_frame_analytics: DataFrameAnalyticsConfig[];
+}
+
+export const ExplorationTitle: React.FC<{ jobId: string }> = ({ jobId }) => (
+  <EuiTitle size="xs">
+    <span>
+      {i18n.translate('xpack.ml.dataframe.analytics.classificationExploration.tableJobIdTitle', {
+        defaultMessage: 'Destination index for classification job ID {jobId}',
+        values: { jobId },
+      })}
+    </span>
+  </EuiTitle>
+);
+
+interface Props {
+  jobId: string;
+  jobStatus: DATA_FRAME_TASK_STATE;
+}
+
+export const ClassificationExploration: FC<Props> = ({ jobId, jobStatus }) => {
+  const [jobConfig, setJobConfig] = useState<DataFrameAnalyticsConfig | undefined>(undefined);
+  const [isLoadingJobConfig, setIsLoadingJobConfig] = useState<boolean>(false);
+  const [jobConfigErrorMessage, setJobConfigErrorMessage] = useState<undefined | string>(undefined);
+  const [searchQuery, setSearchQuery] = useState<ResultsSearchQuery>(defaultSearchQuery);
+
+  const loadJobConfig = async () => {
+    setIsLoadingJobConfig(true);
+    try {
+      const analyticsConfigs: GetDataFrameAnalyticsResponse = await ml.dataFrameAnalytics.getDataFrameAnalytics(
+        jobId
+      );
+      if (
+        Array.isArray(analyticsConfigs.data_frame_analytics) &&
+        analyticsConfigs.data_frame_analytics.length > 0
+      ) {
```

Comment:
What happens if the config isn't part of the response and no error gets thrown?

Reply:
Good point - updating to display a 'No results' message if no results are actually returned. 👍